content/2022/videookulografija.md

```diff
@@ -2,6 +2,19 @@
 title: Videookulografija
 summary: Projekat iz videookulografije rađen na letnjem kampu za stare polaznike 2022. godine od Anje Kovačev i Lenke Milojčević.
 ---
+
+## **Uvod**
+Život u današnjem društvu ima mnogo prednosti u svakom pogledu. Tehnologija napreduje iz dana u dan. Taj brz razvoj zahteva velike potrebe koje ne mogu svi ljudi da ispune. Na ideju za ovaj projekat smo došle uvidevši potrebe osoba sa invaliditetom da komuniciraju, igraju video igre, da koriste kompijuter ili da se kreću. 
+Napretkom tehnologije, VOG (videookulografiju) je zamenila EOG (elektrookulografiju) metodu. Videookulografija je metoda u kojoj se pomoću kamere prate pokreti oka, tj. zenice. U ovom radu proučavan je 2D VOG sistem, što znači da se posmatraju dve ose – horizontalna osa na kojoj se detektuju pokreti na levo i desno, i vertikalna osa na kojoj je prate pogled na gore i dole.
+
+
+2D VOG smatra se jednom od najpouzdanijih metoda, a ima široku primenu. Primenjuje se u oftalmologiji za proučavanje patoloških stanja kao što je nistagmus1 (nistagmus predstavlja ritmičke, nevoljne  trzaje očnih jabučica). VOG koriste kvadriplegičari i osobe sa oštećenjima motornih neurona kao sredstvo za komunikaciju ili kretanje (upravljanje tastaturom i invalidskim kolicima pokretima očiju). Takođe, u industriji se koristi za praćenje reakcija klijenata. Još se koristi za posmatranje reakcije vozača prilikom vožnje, u vojnie svrhe i slično.
```

Comment:
treba li ovaj kec da stoji? Trebaju nam neke reference za sve ovo što pišete jer je očigledno da nije iz glave